model/task/local_service.go

```diff
@@ -116,19 +143,86 @@ func (s *localService) Get(ctx context.Context, taskOpts []Task, fields ...strin
 	}
 
 	var allDBTaskResults []testresult.DbTaskTestResults
-	cur, err := s.env.DB().Collection(testresult.Collection).Find(ctx, filter, opts)
+	cur, err := s.env.CedarDB().Collection(testresult.Collection).Find(ctx, filter, opts)
 	if err != nil {
 		return nil, errors.Wrap(err, "finding DB test results")
 	}
 	if err = cur.All(ctx, &allDBTaskResults); err != nil {
 		return nil, errors.Wrap(err, "reading DB test results")
 	}
-
 	allTaskResults := make([]testresult.TaskTestResults, len(allDBTaskResults))
+
+	if len(fields) == 0 {
```

Comment:
What's the significance of fields being empty? Can we add a comment explaining more?

Reply:
Added a comment explaining.